smoke-tests/matrix/src/payara-dockerfile.template

```diff
@@ -0,0 +1,5 @@
+FROM payara/server-full:@version@
```

Comment:
Can we use docker build-arg instead of custom templating?

https://docs.docker.com/engine/reference/commandline/build/#set-build-time-variables---build-arg

Reply:
Can you use them for FROM? I thought not

Reply:
Think so, I've used it in Zipkin before https://github.com/openzipkin/zipkin/blob/master/docker/Dockerfile#L31

Reply:
https://docs.docker.com/engine/reference/builder/#understand-how-arg-and-from-interact

Reply:
Still struggling to get it working... What's the big difference between gradle token expansion and docker build args?

Reply:
Fixed

Reply:
Just that it can be more convenient to run docker build locally sometimes. But no big deal.

Reply:
oh that's cool, hadn't seen that before